Source/WebCore/accessibility/AXTextMarker.cpp

```diff
@@ -194,7 +194,7 @@ std::optional<BoundaryPoint> AXTextMarker::boundaryPoint() const
     RefPtr node = characterOffset.node;
 
     int offset = characterOffset.startIndex + characterOffset.offset;
-    if (AccessibilityObject::replacedNodeNeedsCharacter(*node) || node->hasTagName(HTMLNames::brTag))
+    if (AccessibilityObject::replacedNodeNeedsCharacter(*node) || WebCore::elementName(node.get()) == ElementName::HTML_br)
```

Comment:
Could use the `Node&` version of `elementName` here (the prior code dereferenced it unconditionally)